src/tokens/tokens.service.ts

```diff
@@ -130,7 +130,8 @@ export class TokensService {
       return { id: response.data.id };
     } else {
       const to: string[] = [];
-      for (let i = 0; i < dto.amount; i++) {
+      const amount = parseInt(dto.amount);
```

Comment:
Are we worried about `amount` being too large for a JavaScript `int`?

What happens if `amount` is not a whole number?

Reply:
👍  good point that this isn't clear. Added a comment, as this code is NFT related.